explorer.js

```diff
@@ -1340,6 +1342,8 @@ $(document).ready(() => {
 	// Default AWS region and v4 signature
 	AWS.config.update({ region: '' });
 	AWS.config.update({ signatureVersion: 'v4' });
+	AWS.config.update({ correctClockSkew: true });
+	AWS.config.correctClockSkew = true;
```

Comment:
Do we really need both? Isn't AWS.config.update({ correctClockSkew: true }) enough?

Reply:
same as above, woule prefer the two usage listed in the doc to be sure we really fix it for all usage

Reply:
> same as above, woule prefer the two usage listed in the doc to be sure we really fix it for all usage

I think config.update is just an alias, if you only do AWS.config.update({ correctClockSkew: true }); and then do console.log(AWS.config); you will notice that AWS.config has a new correctClockSkew property set to true